client/components/theme/index.jsx

```diff
@@ -396,7 +398,7 @@ export class Theme extends Component {
 				}
 			),
 			{
-				Link: <LinkButton isLink onClick={ this.goToCheckout } />,
+				Link: <LinkButton isLink onClick={ () => this.goToCheckout( 'premium' ) } />,
```

Comment:
Since `premium` is the default param, you could also omit it
```suggestion
				Link: <LinkButton isLink onClick={ () => this.goToCheckout() } />,
```

But I guess having an explicit param always is more readable.

Reply:
Yup, it's more readable. :)